.github/workflows/validate.yml

```diff
@@ -59,9 +59,11 @@ jobs:
         run: npm run validate
 
       - name: ⬆️ Upload coverage report
-        uses: codecov/codecov-action@v1
+        uses: codecov/codecov-action@v4
         with:
+          fail_ci_if_error: true 
           flags: ${{ matrix.react }}
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
This will not be available on forks but tokenless uploads are still supported for those:

> Tokenless uploading is unsupported. However, PRs made from forks to the upstream public repos will support tokenless (e.g. contributors to OS projects do not need the upstream repo's Codecov token)

-- https://github.com/codecov/codecov-action#breaking-changes